Examples/Framework/include/ActsExamples/Framework/WhiteBoard.hpp

```diff
@@ -80,6 +71,30 @@ class WhiteBoard {
     const std::type_info& type() const override { return typeid(T); }
   };
 
+  /// Store an holder on the white board and transfer ownership.
```

Comment:
```suggestion
  /// Store a holder on the white board and transfer ownership.
```

does it really transfer ownership?

Reply:
Probably not anymore. Can update the comment

Reply:
```suggestion
  /// Store a holder on the white board.
```